router/budget.go

```diff
@@ -20,16 +20,28 @@ func (rt *Router) handleNewBudget(w http.ResponseWriter, r *http.Request) {
 		log.Fatalf("handleNewBudget: %v\n", err)
 	}
 
+	ies, err := rt.IncomeExpenseRepo.GetAllWithUserID(1)
+	if err != nil {
+		log.Fatalf("handleNewBudget: %v\n", err)
+	}
+
 	data := struct {
-		AppTitle   string
-		PageTitle  string
-		Categories []database.Category
+		AppTitle         string
+		PageTitle        string
+		Categories       []database.Category
+		DefaultAllocated float64
 	}{
 		AppTitle:   AppTitle,
 		PageTitle:  "New Budget",
 		Categories: categories,
 	}
 
+	for _, ie := range ies {
+		if ie.Enabled == true {
```

Comment:
`Enabled` is either true or false, you don't need to compare it to another boolean.

Idiomatically:
```
if !ie.Enabled {
    continue
}
data.DefaultAllocated += ie.amount
```